Functions/BIDS/xASL_bids_Config.m

```diff
@@ -28,22 +28,22 @@
 bidsPar.listFieldsRemoveGeneral = {'ProcedureStepDescription' 'SeriesDescription' 'ProtocolName'... % Anonymization
 	'PhilipsRescaleSlope' 'PhilipsRWVSlope' 'PhilipsScaleSlope' 'PhilipsRescaleIntercept' 'UsePhilipsFloatNotDisplayScaling' 'RWVSlope',...
 	'PhilipsRWVIntercept' 'RescaleSlopeOriginal' 'RescaleSlope' 'MRScaleSlope' 'RescaleIntercept', 'RWVIntercept',... % Scales slopes have to be applied and removed
-	'Modality', 'ImagingFrequency', 'PatientPosition','ImageType','PhaseEncodingPolarityGE',...
-	'SeriesNumber','AcquisitionTime','AcquisitionNumber','SliceThickness','SpacingBetweenSlices','SAR',...
+	'Modality', 'ImagingFrequency', 'PatientPosition','ImageType','PhaseEncodingPolarityGE','EstimatedEffectiveEchoSpacing','EstimatedTotalReadoutTime',...
+	'SeriesNumber','AcquisitionTime','AcquisitionNumber','SliceThickness','SpacingBetweenSlices','SAR','InternalPulseSequenceName','PhaseEncodingStepsNoPartialFourier', 'PartialFourierEnabled',...
 	'Rows' 'Columns' 'InPlanePhaseEncodingDirection' 'NumberOfTemporalPositions','GELabelingType', 'PulseSequenceName','AcquisitionContrast',...
 	'PhilipsNumberTemporalScans','TemporalPositionIdentifier','PhilipsLabelControl','NumberOfTemporalPositions','PhoenixProtocol','PhoenixAnalyzed','SiemensSliceTime',...
 	'PercentPhaseFOV','AcquisitionMatrixPE','ReconMatrixPE','PixelBandwidth','ImageOrientationPatientDICOM','ComplexImageComponent',...
 	'InPlanePhaseEncodingDirectionDICOM','ConversionSoftware','ConversionSoftwareVersion','AcquisitionMatrix','GELabelingDuration',...
-	'EchoTrainLength','PhaseEncodingSteps','BodyPartExamined','ShimSetting','TxRefAmp','PhaseResolution','InstanceNumber',...
-	'RefLinesPE','BandwidthPerPixelPhaseEncode','ImageComments','ConsistencyInfo','WipMemBlock','Interpolation2D','TotalAcquiredVolumes',...
-	'SaturationStopTime','BaseResolution','DerivedVendorReportedEchoSpacing','RawImage','PhaseOversampling','BolusDuration','scaleFactor','EchoNumber'}; % Fields to exclude as not defined in BIDS
+	'EchoTrainLength','PhaseEncodingSteps','BodyPartExamined','ShimSetting','TxRefAmp','PhaseResolution','InstanceNumber', 'WaterFatShift',...
+	'RefLinesPE','BandwidthPerPixelPhaseEncode','ImageComments','ConsistencyInfo','WipMemBlock','Interpolation2D','TotalAcquiredVolumes', 'NumberOfExcitations', 'PrescanReuseString', ...
+	'SaturationStopTime','BaseResolution','DerivedVendorReportedEchoSpacing','RawImage', 'T1', 'PercentSampling', 'ImageOrientationText', 'PhaseOversampling','BolusDuration','scaleFactor','EchoNumber'}; % Fields to exclude as not defined in BIDS
 
 % A list of fields to remove the ASL-BIDS JSON files only
 bidsPar.listFieldsRemoveASL = {'SliceReadoutTime','RepetitionTime','InversionTime','LabelOffset','PostLabelDelay','NumRFBlocks',...
 	'GELabelingDuration','RFGap','MeanGzx10','PhiAdjust','M0','LabelingType','ScanType','SequenceType'}; % Fields to exclude from ASL only
 
 % A list of fields to remove the ASL-M0 JSON files only
```

Comment:
to remove from? Otherwise unclear.

Why do we remove fields here, because we copied the `asl.json` to `m0.json`? Or because Chris Rorden outputs too many fields? (Please clarify this here in a comment)

Reply:
Yes - there are far too many fields that don't exist in BIDS, so we remove them. Explained in a comment - see 43d03f7b